### PR TITLE
clamav: Add version 0.102.4

### PR DESCRIPTION
**Pull request by ouyangwanxi**

### CODE REVIEW

File: bucket/clamav.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "version": "0.102.4",
+    "description": "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats.",
+    "homepage": "https://www.clamav.net/",
+    "license": "GNU General Public License v2",
```
Comment (Ash258) [inline bucket/clamav.json:5]: Invalid SPDX identified

File: bucket/clamav.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
+    "version": "0.102.4",
+    "description": "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats.",
+    "homepage": "https://www.clamav.net/",
```
Comment (Ash258) [inline bucket/clamav.json:4]: Slash is not needed

image

File: bucket/clamav.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "clambc.exe",
+        "clamconf.exe",
+        "clamd.exe",
+        "clamdscan.exe",
+        "clamscan.exe",
+        "clamsubmit.exe",
+        "freshclam.exe",
+        "sigtool.exe"
+    ],
+    "pre_install": [
```
Comment (Ash258) [inline bucket/clamav.json:26]: Pre_install happen before binary creation.

Move it before bin property

File: bucket/clamav.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        "Set-Content \"$dir\\clamd.conf\" '#' -Encoding Ascii",
+        "Set-Content \"$dir\\freshclam.conf\" '#' -Encoding Ascii"
```
Comment (Ash258) [inline bucket/clamav.json:28]: Check existence of already persited files

File: bucket/clamav.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+    ],
+    "persist": [
+        "conf_examples",
```
Comment (Ash258) [inline bucket/clamav.json:32]: File or folder?

Why are examples persited?

Comment (ouyangwanxi) [reply bucket/clamav.json:31]: folder
Users need to copy and modify the files

File: bucket/clamav.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
+{
+    "version": "0.102.4",
+    "description": "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats.",
+    "homepage": "https://www.clamav.net",
+    "license": "GPLv2",
```
Comment (Ash258) [inline bucket/clamav.json:5]: Invalid SPDX identifier

File: bucket/clamav.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+            "hash": "b564cf32e6c44b39dfd7c9f7719c62f621f40db5ae67f37e2634b92827c0d129"
+        },
+        "64bit": {
+            "url": "https://www.clamav.net/downloads/production/clamav-0.102.4-win-x64-portable.zip",
+            "hash": "fbe8b74d719983b2732f4a5629cf5ca813f5095c1d8d1aea36ef3737f5c78878"
+        }
+    },
+    "pre_install": [
+        "Set-Content \"$dir\\clamd.conf\" '#' -Encoding Ascii",
+        "Set-Content \"$dir\\freshclam.conf\" '#' -Encoding Ascii"
```
Comment (Ash258) [inline bucket/clamav.json:18]: Check already persisted files

File: bucket/clamav.json [hunk 8]
```diff
@@ -0,0 +1,50 @@
+{
+    "version": "0.102.4",
+    "description": "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats.",
```
Comment (Ash258) [inline bucket/clamav.json:3]: - Do not mention application name when it is same as manifest
- `Open source` is clear from license

File: bucket/clamav.json [hunk 9]
```diff
@@ -0,0 +1,50 @@
+{
+    "version": "0.102.4",
+    "description": "An open source antivirus engine for detecting trojans, viruses, malware & other malicious threats.",
```
Comment (Ash258) [inline bucket/clamav.json:3]: https://github.com/lukesampson/scoop-extras/pull/4581#discussion_r470688617

**Pull request closed without merging**